File: Python Exams/Basic Exam Collection/Basics Online Exam - 20 and 21 April 2019/03. Painting Eggs/03. Painting Eggs.py
size = input()
color = input()
count = int(input())

eggs_dict = {
    'Large' : {'Red' : 16, 'Green' : 12, 'Yellow' : 9},
    'Medium' : {'Red' : 13, 'Green' : 9, 'Yellow' : 7},
    'Small' : {'Red' : 9, 'Green' : 8, 'Yellow' : 5},
}

total_sum = eggs_dict[size][color] * count * 0.65
print(f'{total_sum:.2f} leva.')




'''
Large
Red
7
'''

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 20 and 21 April 2019/03. Painting Eggs/03. Painting Eggs.js
function paintingEggs(size, color,  count ) {
    
    const eggsObj = {
        'Large' : {'Red' : 16, 'Green' : 12, 'Yellow' : 9},
        'Medium' : {'Red' : 13, 'Green' : 9, 'Yellow' : 7},
        'Small' : {'Red' : 9, 'Green' : 8, 'Yellow' : 5},
    }
    
    let totalSum = eggsObj[size][color] * count * 0.65
    
    return `${totalSum.toFixed(2)} leva.`
}

//console.log(paintingEggs('Large', 'Red', 7))



